app/_locales/en/messages.json

```diff
@@ -594,7 +594,8 @@
     "message": "Decrypt request"
   },
   "defaultNetwork": {
-    "message": "The default network for Ether transactions is Mainnet."
+    "message": "The default network for Ether transactions is Mainnet. You can also $1 testnets $2.",
+    "description": "$1 is the 'enable' or 'disable' key, depending on whether the display of test networks is enabled or not. $2 is a clickable link with text defined by the 'here' key. The link will open to the advanced settings where users can enable the display of test networks in the network dropdown."
```

Comment:
Hmmm.....`You can also $1 testnets $2.` won't appear in the translations when we ship :thinking: 

So should we include it in english in other languages?

Reply:
Sounds good to me, this translation will just fallback to english now